makefiles/golang.mk

```diff
@@ -26,7 +26,7 @@ ifneq ($(wildcard $(BUILD_WORK)/golang/.build_complete),)
 golang:
 	@echo "Using previously built golang."
 else
-golang: golang-setup
+golang: golang-setup uikittools
```

Comment:
It doesn't actually depend on `uikitools`, packages built with go *may* depend on uikittools.